src/nodes.coffee

```diff
@@ -3175,6 +3201,10 @@ exports.ImportDeclaration = class ImportDeclaration extends ModuleDeclaration
     code.push @makeCode ';'
     code
 
+  astNode: (o) ->
+    o.importedSymbols = []
```

Comment:
Why is this here?

Reply:
This `o.importedSymbols` gets used to track imported names eg for throwing an error if you try and import the same name more than once (eg `import {foo, foo} from 'bar'`) in `ImportSpecifier::addIdentifierToScope()`

Reply:
I see, you’re duplicating what’s in `compileNode`. I guess we can’t just put this in the constructor to avoid the duplication?

I also don’t care all that much, it’s one line.

Reply:
Ya in general I took the approach of trying to DRY things up by sharing small helper methods between the `compileNode()` path and the AST generation path, but here this didn't seem worth trying to encapsulate